domain_access/src/DomainAccessManager.php

```diff
@@ -66,6 +67,7 @@ public function getAccessValues(EntityInterface $entity, $field_name = DOMAIN_AC
    * @inheritdoc
    */
   public function getAllValue(EntityInterface $entity) {
+    /** @var FieldableEntityInterface $entity */
```

Comment:
I believe the same response applies - currently the code will fail if ever passed an entity that is not fieldable. This type hint exposes that problem. But there may be better ways to solve the problem - but choosing among them is beyond what I can do with my understanding of the module.